include/sysexits.h

```diff
@@ -35,9 +35,10 @@
 /*
  *  SYSEXITS.H -- Exit status codes for system programs.
  *
- *	This include file attempts to categorize possible error
- *	exit statuses for system programs, notably delivermail
- *	and the Berkeley network.
+ *	This include file attempts to categorize 4BSD error exit
+ *	statuses for system programs, notably delivermail (which
+ *	became sendmail) and the Berkeley network. These values
+ *	have been deprecated since FreeBSD 8.0, see style(9).
```

Comment:
s/deprecated since/deprecated in the base system since/

Reply:
I think that's especially nice since it folds "see style(9)" onto it's own line.

Reply:
I put it here because this comment sounded pretty sane if these values are deprecated.
https://github.com/freebsd/freebsd-src/pull/1161#issuecomment-2067726470